usr/src/man/man1m/getent.1m

```diff
@@ -19,7 +19,7 @@
 .\" Copyright (c) 2014 Gary Mills
 .\" Copyright (c) 2018 Peter Tribble
```

Comment:
No there was not I had some problems because of newer nexenta copyright and in those files I left the newer one and dropped the copyright from the original patch as that was already covered by the newer copyright statement. But in this file there was none. I did update the manpage date however.